charts/pravega-operator/templates/certificate.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     secretName: {{ template "pravega-operator.fullname" . }}-key-pair
 {{- end }}
 ---
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: cert-manager.io/v1alpha2
 kind: Certificate
 metadata:
   name: {{ template "pravega-operator.fullname" . }}-selfsigned-cert
```

Comment:
Are we planning to change pravega-operator full name , so that by adding `selfsigned-cert-tls` it should not exceed 63 characters

Reply:
The possibility for this scenario to occur is rare, and if the need is felt, will make this change for all the charts, but for now I think this should work